app/assets/javascripts/views/story_view.js

```diff
@@ -546,6 +546,7 @@ module.exports = FormView.extend({
 
       this.renderNotes();
 
+      this.renderReactComponents();
```

Comment:
@andrerpbts actually it was my idea to have this `renderReactComponents` here haha. The idea is exactly suggesting that all react rendering inside this view should be inside this method (it won't get too big because it'll just find the right DOM node and pass the props) so with the time we'll grow the `renderReactComponents` method and shrink the `render` method until we replace the whole view  built with jQuery/EJS with React instead. What do you think?